com.unity.render-pipelines.high-definition/Documentation~/Override-Volumetric-Clouds.md

```diff
@@ -118,6 +118,7 @@ When importing these two map textures, disable **sRGB**. For best results, do no
 | **Property**                     | **Description**                                              |
 | -------------------------------- | ------------------------------------------------------------ |
 | **Temporal Accumulation Factor** | The amount of temporal accumulation to apply to the clouds. Temporal accumulation increases the visual quality of clouds by decreasing the noise. A higher value produces better quality clouds, but can create [ghosting](Glossary.md#ghosting). |
+| **Ghosting Reduction**           | Indicates whether to remove the ghosting induced by the temporal accumulation. When active, it may introduce some flickering at lower temporal accumulation factors. |
```

Comment:
| **Ghosting Reduction**           | When you enable this property, HDRP removes the ghosting caused by temporal accumulation. This effect may cause a flickering effect when the **Temporal Accumulation Factor** value is low. |